src/components/Homepage.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable global-require */
+/* eslint-disable max-len */
+import homepageStyles from './Homepage.module.css';
+
+// eslint-disable-next-line arrow-body-style
```

Comment:
- [ ] Let's don't disable linter rules. These rules are here to help us write the best code.